addRate.js

```diff
@@ -35,4 +35,10 @@ $(document).ready(function() {
       },
     },
   });
+
+  $('.userDetails').append(
```

Comment:
let's use a jquery variable for that? i think i saw you're using the same selector in the other pr..